x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/components/setup_technology_selector.tsx

```diff
@@ -8,121 +8,110 @@
-                id="xpack.fleet.setupTechnology.agentlessDrowpownDescription"
-                defaultMessage="Set up the integration without an agent"
-              />
-            </p>
-          </EuiText>
-        </>
-      ),
-    },
-    {
-      value: SetupTechnology.AGENT_BASED,
-      inputDisplay: (
-        <FormattedMessage
-          id="xpack.fleet.setupTechnology.agentbasedInputDisplay"
-          defaultMessage="Agent-based"
-        />
-      ),
-      dropdownDisplay: (
-        <>
-          <strong>
-            <FormattedMessage
-              id="xpack.fleet.setupTechnology.agentbasedDrowpownDisplay"
-              defaultMessage="Agent-based"
-            />
-          </strong>
-          <EuiText size="s" color="subdued">
-            <p>
-              <FormattedMessage
-                id="xpack.fleet.setupTechnology.agentbasedDrowpownDescription"
-                defaultMessage="Set up the integration with an agent"
-              />
-            </p>
-          </EuiText>
-        </>
-      ),
-    },
-  ];
-
   return (
-    <>
-      <EuiSpacer size="l" />
-      <EuiFormRow
-        fullWidth
-        label={
+    <EuiDescribedFormGroup
+      title={
+        <h3>
           <FormattedMessage
             id="xpack.fleet.setupTechnology.setupTechnologyLabel"
-            defaultMessage="Setup technology"
+            defaultMessage="Deployment options"
           />
-        }
-      >
-        <EuiSuperSelect
-          disabled={disabled}
-          options={options}
-          valueOfSelected={setupTechnology}
-          placeholder={
-            <FormattedMessage
-              id="xpack.fleet.setupTechnology.setupTechnologyPlaceholder"
-              defaultMessage="Select the setup technology"
-            />
-          }
-          onChange={onSetupTechnologyChange}
-          itemLayoutAlign="top"
-          hasDividers
-          fullWidth
-          data-test-subj={SETUP_TECHNOLOGY_SELECTOR_TEST_SUBJ}
+        </h3>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.fleet.setupTechnology.setupTechnologyDescription"
+          defaultMessage="Select a deployment mode for this integration."
         />
-      </EuiFormRow>
-    </>
+      }
+    >
+      <EuiRadioGroup
+        disabled={disabled}
+        name="SetupTechnologySelector"
+        data-test-subj={SETUP_TECHNOLOGY_SELECTOR_TEST_SUBJ}
+        options={[
+          {
+            id: `SetupTechnologySelector_${SetupTechnology.AGENTLESS}`,
+            value: SetupTechnology.AGENTLESS,
+            disabled: !allowedSetupTechnologies.includes(SetupTechnology.AGENTLESS),
+            label: (
+              <>
+                <strong>
+                  <FormattedMessage
+                    id="xpack.fleet.setupTechnology.agentlessInputDisplay"
+                    defaultMessage="Agentless"
+                  />{' '}
+                  <EuiBetaBadge
+                    label="Beta"
```

Comment:
a bit unrelated, I'm wondering when the Beta badge is planned to be removed?

Reply:
it was already removed for the CSPM editor in https://github.com/elastic/kibana/pull/208314

I'll check around to see if it can be removed across the board

Reply:
closing the loop here, agentless is not GA outside of CSPM, GA work is being tracked in https://github.com/elastic/security-team/issues/11353